tests/integration/targets/docker_network/tasks/tests/overlay.yml

```diff
@@ -60,3 +60,55 @@
     docker_swarm:
       state: absent
       force: true
+
+####################################################################
+## ingress #########################################################
+####################################################################
+
+- block:
+  # Overlay networks require swarm initialization before they'll work
+  - name: swarm
+    docker_swarm:
+      state: present
+      advertise_addr: "{{ ansible_default_ipv4.address | default('127.0.0.1') }}"
+
+  - name: ingress
+    docker_network:
+      name: "{{ nname_1 }}"
+      driver: overlay
+      ingress: true
```

Comment:
This task currently fails with
```
01:43 An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ansible_collections.community.docker.plugins.module_utils._api.errors.APIError: 409 Client Error for http+docker://localhost/v1.43/networks/create: Conflict ("rpc error: code = AlreadyExists desc = ingress network (j14ca3x29squ0lvqcyooyr5ik) is already present")
```

Reply:
Fixed in a95263198286ee13d5a4e86f57e85b8e69a0544c